.gitpod.yml

```diff
@@ -0,0 +1,2 @@
+tasks:
+- init: ./gradlew build
```

Comment:
any reason not to call  `./1-gradle-build.sh`

Reply:
Nope, didn't know it existed. Just followed the instructions from `README.MD`.